frontend/www/js/omegaup/components/problem/SettingsSummary.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div class="panel">
+    <h1 class="title">
+      {{ title }}
+      <template v-if="showVisibilityIndicators">
+        <img
+          src="/media/quality-badge-sm.png"
+          v-bind:title="T.wordsHighQualityProblem"
+          v-if="problem.quality_seal || problem.visibility === 3"
+        />
+        <span
+          class="glyphicon glyphicon-warning-sign"
+          v-bind:title="T.wordsWarningProblem"
+          v-if="problem.visibility === 1 || problem.visibility === -1"
+        ></span>
+        <span
+          class="glyphicon glyphicon-eye-close"
+          v-bind:title="T.wordsPrivate"
+          v-if="problem.visibility === 0 || problem.visibility === -1"
+        ></span>
+        <span
+          class="glyphicon glyphicon-ban-circle"
+          v-bind:title="T.wordsBannedProblem"
+          v-if="problem.visibility <= -2"
+        ></span>
+      </template>
+      <template v-if="showEditLink">
+        (<a href="/problem/{$problem_alias}/edit/">{{ T.wordsEdit }}</a
+        >)
+      </template>
+    </h1>
+    <table>
+      <tr>
+        <th scope="row">{{ T.wordsPoints }}</th>
+        <td>{{ problem.points }}</td>
+        <th scope="row">{{ T.arenaCommonMemoryLimit }}</th>
+        <td data-memory-limit>{{ memoryLimit }}</td>
+      </tr>
+      <tr>
+        <th scope="row">{{ T.arenaCommonTimeLimit }}</th>
+        <td>{{ timeLimit }}</td>
+        <th scope="row">{{ T.arenaCommonOverallWallTimeLimit }}</th>
+        <td>{{ overallWallTimeLimit }}</td>
+      </tr>
+      <tr>
+        <template v-if="!showVisibilityIndicators">
+          <th scope="row">{{ T.wordsInOut }}</th>
+          <td>{{ T.wordsConsole }}</td>
+        </template>
+        <th scope="row">{{ T.problemEditFormInputLimit }}</th>
+        <td>{{ inputLimit }}</td>
+      </tr>
+    </table>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.panel {
+  .title {
+    text-align: center;
+    font-size: 1.5em;
+    margin: 1em;
+  }
+  table {
+    width: 30em;
+    margin: 10px auto;
+    td {
+      text-align: center;
+      border: 1px solid #000;
+      padding: 2px;
+    }
+    th[scope='row'] {
+      border: 1px solid #000;
+      padding: 2px;
+      font-weight: bold;
+    }
```

Comment:
nit: así se evita repetir cosas

```suggestion
    td {
      text-align: center;
    }
    th[scope='row'] {
      font-weight: bold;
    }
    td, th[scope='row'] {
      border: 1px solid #000;
      padding: 2px;
    }
```

Reply:
Listo